draft-ietf-sframe-enc.md

```diff
@@ -1299,65 +1427,65 @@ In audio streams, there is typically a one-to-one relationship between frames
 and packets, so the overhead is the same whether one uses SFrame at a per-packet
 or per-frame level.
 
-The below table considers three scenarios, based on recommended configurations
+{{audio-overhead}} considers three scenarios that are based on recommended configurations
 of the Opus codec {{?RFC6716}}:
 
-* Narrow-band speech: 120ms packets, 8Kbps
-* Full-band speech: 20ms packets, 32Kbps
-* Full-band stereo music: 10ms packets, 128Kbps
+* Narrow-band (NB) speech: 120 ms packets, 8 kbps
+* Full-band (FB) speech: 20 ms packets, 32 kbps
+* Full-band stereo music: 10 ms packets, 128 kbps
 
-| Scenario                 | fps | Base Kbps | Overhead Kbps | Overhead % |
-|:-------------------------|:---:|:---------:|:-------------:|:----------:|
-| NB speech, 120ms packets | 8.3 |         8 |           1.4 |      17.9% |
-| FB speech, 20ms packets  |  50 |        32 |           8.6 |      26.9% |
-| FB stereo, 10ms packets  | 100 |       128 |          17.2 |      13.4% |
-{: #audio-overhead title="SFrame overhead for audio streams" }
+| Scenario                  | Frames per Second (fps) | Base kbps | Overhead kbps | Overhead % |
+|:--------------------------|:---:|:---------:|:-------------:|:----------:|
+| NB speech, 120 ms packets | 8.3 |         8 |           1.4 |      17.9% |
+| FB speech, 20 ms packets  |  50 |        32 |           8.6 |      26.9% |
+| FB stereo, 10 ms packets  | 100 |       128 |          17.2 |      13.4% |
+{: #audio-overhead title="SFrame Overhead for Audio Streams" }
```

Comment:
TODO(RLB): Add "fps" terminology above, revert expansion, and realign column dividers.